pwa/src/templates/sidebar/Sidebar.module.css

```diff
@@ -30,4 +30,5 @@
   max-width: 100%;
   max-height: 100%;
   object-fit: contain;
+  cursor: url(), pointer;
```

Comment:
Very good,
But as seen on line 16 in this file `cursor: pointer;` works as well. We dont need to include a file so the `url()` function is not necessary.